packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -26,13 +26,6 @@ export const videoGalleryContainerStyle: IStackStyles = {
   root: { position: 'relative', height: '100%', width: '100%', padding: '0.5rem' }
 };
 
-/**
- * @private
- */
-export const videoGalleryContainerTokens = {
-  childrenGap: '0.5rem'
-};
```

Comment:
I think we still need this gap when the HorizontalGallery is present otherwise it looks like this:
![image](https://user-images.githubusercontent.com/79475487/145144415-77230597-29c0-46ed-a0f9-4d97a988bce3.png)

Reply:
Solved it by adding a top padding to horizontal gallery.
The reason I needed to remove the children gap token was because it was adding an undesired padding to the layerhost causing a shift in the local video placement. This  was messing up the whole layout.

Reply:
> Let's keep this PR out of release (OK to land on `main`).
> 
> Could we size and position the `LayerHost` such that there's margin equal to the size of the PIP modal? In that way, the PIP will be draggable such that it never leaves the `VideoGallery` boundary at all.

That would cause the pip modal to shift towards the centre of video gallery and dragging it out will cause it to clip.